test/internals/toString.test.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const Fastify = require('../..')
+const fp = require('fastify-plugin')
+
+test('toString method should exist', t => {
+  t.plan(1)
+  const fastify = Fastify()
+  t.ok(fastify.toString)
+})
+
+test('should return valid string', t => {
+  t.plan(1)
+  const fastify = Fastify()
+  fastify.get('/', (request, reply) => reply.send({ text: '/' }))
+  fastify.post('/test', (request, reply) => reply.send({ text: 'test' }))
+
+  fastify.use(function helloWorld (instance, req, res) {
+    console.log('Hello, World!')
+  })
+
+  fastify.use('/abc', function routeABC (instance, req, res) {
+    console.log('ABC')
+  })
+
+  fastify.register(fp((instance, opts, next) => {
+    instance.decorate('anonymousPlugin', (a, b) => a + b)
+    next()
+  }))
```

Comment:
If a name it's not provided, we might want to extract the file:line when `fp`  is called (it'd be a PR to fastify-plugin).

Reply:
Hmm, okay I'll see what I come up with. My initial thought would be to use something like `function.caller` but that is not available in strict mode.